File: Selenium/Activities/Activity5.py
from selenium import webdriver
from selenium.webdriver.common.by import By
from selenium.webdriver.firefox.service import Service as FirefoxService
from selenium.webdriver.firefox.webdriver import WebDriver
from selenium.webdriver.support import expected_conditions
from webdriver_manager.firefox import GeckoDriverManager

# setup Firefoxdriver with WebDriverManager
service = FirefoxService(GeckoDriverManager().install())

# start the driver
with webdriver.Firefox(service=service) as driver:
    # Access the URL
    driver.get("https://v1.training-support.net/selenium/input-events")
    # print header of the page
    print("Title of the home page is : ", driver.title)
    # setup the object of Action class
    actions = webdriver.ActionChains(driver)
    # Perform left click
    actions.click().pause(10).perform()
    frontText = driver.find_element(By.CLASS_NAME,"active")
    print("Front after left click is: ",frontText.text)
    #Perform double click
    actions.double_click().pause(10).perform()
    randomSide = driver.find_element(By.CLASS_NAME,"active")
    print("Front after double click is: ",randomSide.text)
    #perform right click
    actions.context_click().pause(10).perform()
    frontTextRight = driver.find_element(By.CLASS_NAME,"active")
    print("Front after right click is: ",frontTextRight.text)
    #Close the browser
    driver.close()

File: Selenium/Activities/Activity5.java
package Activities;

import io.github.bonigarcia.wdm.WebDriverManager;
import org.openqa.selenium.By;
import org.openqa.selenium.WebDriver;
import org.openqa.selenium.firefox.FirefoxDriver;
import org.openqa.selenium.interactions.Action;
import org.openqa.selenium.interactions.Actions;

public class Activity5 {
    public static void main(String[] args) throws InterruptedException {
        //setting up system property
        System.setProperty(FirefoxDriver.SystemProperty.BROWSER_LOGFILE,"/dev/null");
        //setting up Webdrivermanager
        WebDriverManager.firefoxdriver().setup();
        //Initializing webdriver
        WebDriver driver = new FirefoxDriver();
        //Access the URL
        driver.get("https://v1.training-support.net/selenium/input-events");
        //print header of the page
        System.out.println("Title of the home page is : "+driver.getTitle());
        //initializing object of Action class
        Actions builder = new Actions(driver);
        //Perform left click
        builder.click().pause(10).build().perform();
      //  Thread.sleep(5000);
        String frontText = driver.findElement(By.className("active")).getText();
        System.out.println("Front after left click is: "+frontText);
        //Perform double click
        builder.doubleClick().pause(10).build().perform();
       // Thread.sleep(5000);
        String randomSide = driver.findElement(By.className("active")).getText();
        System.out.println("Front after double click is: "+randomSide);
        //perform right click
        builder.contextClick().pause(10).build().perform();
       // Thread.sleep(5000);
        String frontTextRight = driver.findElement(By.className("active")).getText();
        System.out.println("Front after right click is: "+frontTextRight);
        //Close the browser
        driver.close();

    }
}
